docs/pages/versions/unversioned/sdk/permissions.md

```diff
@@ -52,23 +52,23 @@ Often you want to be able to test what happens when a user rejects a permission,
 
 The following table shows you which permissions correspond to which packages.
 
-| Permission type             | Packages                                  |
-| --------------------------- | ----------------------------------------- |
-| `NOTIFICATIONS`             | `expo-notifications`                      |
-| `USER_FACING_NOTIFICATIONS` | `expo-notifications`                      |
-| `LOCATION`                  | `expo-location`                           |
-| `CAMERA`                    | `expo-barcode-scanner` <br /> `expo-camera` <br /> `expo-face-detector` <br /> `expo-image-picker` <br /> `expo-media-library` |
-| `AUDIO_RECORDING`           | `expo-av`                                 |
-| `CONTACTS`                  | `expo-contacts`                           |
-| `CAMERA_ROLL`               | `expo-image-picker`<br /> `expo-media-library` |
-| `CALENDAR`                  | `expo-calendar`                           |
-| `REMINDERS`                 | `expo-calendar`                           |
-| `SYSTEM_BRIGHTNESS`         | `expo-brightness`                         |
-| `MOTION`                    | `expo-sensors`                            |
+| Permission type                                  | Packages                                                                                                                       |
+| ------------------------------------------------ | ------------------------------------------------------------------------------------------------------------------------------ |
+| `NOTIFICATIONS`                                  | `expo-notifications`                                                                                                           |
+| `USER_FACING_NOTIFICATIONS`                      | `expo-notifications`                                                                                                           |
+| `LOCATION`                                       | `expo-location`                                                                                                                |
+| `CAMERA`                                         | `expo-barcode-scanner` <br /> `expo-camera` <br /> `expo-face-detector` <br /> `expo-image-picker` <br /> `expo-media-library` |
+| `AUDIO_RECORDING`                                | `expo-av`                                                                                                                      |
+| `CONTACTS`                                       | `expo-contacts`                                                                                                                |
+| `MEDIA_LIBRARY`<br /> `WRITE_ONLY_MEDIA_LIBRARY` | `expo-image-picker`<br /> `expo-media-library`                                                                                 |
```

Comment:
```suggestion
| `MEDIA_LIBRARY`<br /> `MEDIA_LIBRARY_WRITE_ONLY` | `expo-image-picker`<br /> `expo-media-library`                                                                                 |
```
After a small inner struggle I believe that'a a better name for this permission 🤔